test/13_loki-backend.ts

```diff
@@ -60,27 +60,6 @@ export default () => {
 			expect(history.at(-1)).to.deep.equal(convertToOutputLog(log));
 		});
 
-		it('should convert multiple logs with different labels to streams and then back to logs', function () {
```

Comment:
I deleted this test as it no longer makes sense and tests the internals (which no longer exist), so it makes more sense to test via the published/supported APIs